mixin/dashboards.libsonnet

```diff
@@ -0,0 +1,97 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+local logslib = import 'logs-lib/logs/main.libsonnet';
```

Comment:
Is this used anywhere?

Reply:
Thanks for the catch, went ahead and removed it and the dependency :)